cli/src/commands/run.rs

```diff
@@ -53,17 +303,84 @@ pub fn cmd_run(ui: &mut Ui, command: &CommandHelper, args: &RunArgs) -> Result<(
     let _resolved_commits: Vec<_> = workspace_command
         .parse_union_revsets(&args.revisions)?
         .evaluate_to_commits()?
+    let resolved_commits: Vec<_> = workspace_command
+        .parse_revset(&args.revisions)?
+        .evaluate_to_commits()?
         .try_collect()?;
     // Jobs are resolved in this order:
     // 1. Commandline argument iff > 0.
     // 2. the amount of cores available.
     // 3. a single job, if all of the above fails.
-    let _jobs = match args.jobs {
+    let jobs = match args.jobs {
```

Comment:
issue: Isn't the convention that `0` means "use all available parallelism"? I think we should do that instead of producing an error here.

Reply:
See #4149